Adds documentation for AI Credit Limits (#38668)
DataDog/documentation: content/en/account_management/billing/ai_credit_limits.md

---

> title: Setting AI Credit Limits

Reviewer: Sibling billing pages use noun-phrase titles: "AI Credits," "Usage Attribution," "Product Allotments," "Pricing." A gerund title is off-pattern for a reference page.

```suggestion
title: AI Credit Limits
```

description: "Set monthly AI Credit limits at the organization level, the per-user level, or both, and override limits for individual users."
---

## Overview

Organization admins can set monthly caps on AI Credit usage at the organization level, the per-user level, or both, with overrides for individual users. AI Credits are shared across [Bits Chat][1], [Bits Investigation][2], [Bits Code][3], and [Bits Agent Builder][4].

## Permissions

> To view and set AI Credit limits, a user needs the following permissions:

Reviewer: Two things here:

1. "a user needs the following permissions" is ambiguous about whether all three are required or any one of them. Can you confirm which?
2. Docs convention is to link permission slugs to the RBAC permissions reference (see `account_management/billing/_index.md`, link 8).

```suggestion
To view and set AI Credit limits, a user needs all of the following [permissions][6]:
```

This also needs a new link definition at the bottom of the file:

```
[6]: /account_management/rbac/permissions/
```

- `user_access_manage`
- `billing_edit`
- `org_management`

## Where to set limits

> AI Credit limits are configured in [**Bits AI > AI Credits Management**][5].

Reviewer: Re-raising from the last round, since this thread didn't get a reply.

This says limits are configured in **Bits AI > AI Credits Management**, but [`ai_credits.md`](https://github.com/DataDog/documentation/blob/master/content/en/account_management/billing/ai_credits.md) says "All AI Credit management lives in **Plan & Usage > AI Credits**." Both link to the same URL (`app.datadoghq.com/billing/ai-credits`), so one of the two breadcrumbs is stale.

Which is the actual in-app path? I'm happy to fix whichever page is wrong.

Author: this link should actually be https://app.datadoghq.com/bits-ai/ai-credits-management

## Types of limits

You can configure three types of monthly limits:

| Limit type | Description |
|---|---|

> | Org-wide limit | Caps total AI Credit usage across the organization for the month. No org-wide limit is set by default. |

Reviewer: The page alternates between "org-wide," "org," and "organization" (lines 28, 36, 42, 45, 48). "Org" is informal so we should use "organization" or "organization-wide" consistently throughout. 

```suggestion
| Organization-wide limit | Caps total AI Credit usage across the organization for the month. No organization-wide limit is set by default. |
```

| Default per-user limit | Caps monthly AI Credit usage for each user who doesn't have an individual override. |
| Per-user override | Sets a custom monthly limit for an individual user, replacing the default per-user limit for that user. |

## How limits are applied

> - When multiple limits are configured, the most restrictive applicable limit applies to a user.

Reviewer: Re-raising from the last round, since this thread didn't get a reply. This is the one I'd most like resolved before merge.

Three statements on this page describe three different behaviors for the same situation:

- Line 30: a per-user override "replac[es] the default per-user limit for that user"
- Line 34: "the most restrictive applicable limit applies"
- Line 35: "the higher of the two applies"

These can't all be true. The practical risk: an admin sets an override *lower* than the default expecting it to bind, and by line 35 it doesn't. That's a billing-behavior claim people will act on.

Once you confirm the real precedence, I can rewrite these three lines so they agree.

Author: This should read:

When both an organization limit and a user limits (or override) are configured, the most restrictive applicable limit applies to a user.

> - If a user has both a default per-user limit and an override configured, the higher of the two applies, and the user's usage is still subject to the org-wide limit.

Reviewer: Small leftover from the terminology pass: this is one of two remaining uses of "org" (the other is line 47). (Suggestion below is Claude-authored.)

```suggestion
- If a user has both a default per-user limit and an override configured, the higher of the two applies, and the user's usage is still subject to the organization-wide limit.
```

Note this line is also covered by the precedence question above, so it may get rewritten anyway.

- A configured org-wide limit acts as the ceiling for the organization, regardless of the sum of per-user limits.

> - Raising a limit unblocks users who had reached the previous limit. Lowering a limit blocks users whose usage already exceeds the new value.

Reviewer: Tense shift to past perfect mid-list.

```suggestion
- Raising a limit unblocks users who reached the previous limit. Lowering a limit blocks users whose usage already exceeds the new value.
```

- When a user or the organization reaches a limit, affected users can't use Bits Chat, Bits Investigation, Bits Code, or Bits Agent Builder, and a banner shows the reset date.

### Examples

> | Org limit | Per-user limit | Users | Max hypothetical usage | Credits charged if limits are reached | Result |

Reviewer: Two header tweaks: spell out "Maximum," and "Credits charged if limits are reached" reads as though credits are charged *because* a limit was hit.

```suggestion
| Organization limit | Per-user limit | Users | Maximum hypothetical usage | Maximum billable Credits | Result |
```

|---|---|---|---|---|---|

> | 2,000 Credits | 50 Credits per user | 20 | 50 × 20 = 1,000 Credits | 1,000 Credits | Each user can use up to 50 AI Credits without other restrictions. |

Reviewer: Terminology consistency: the tables use bare capitalized "Credits" while the prose uses "AI Credits." The existing [AI Credits page](https://github.com/DataDog/documentation/blob/master/content/en/account_management/billing/ai_credits.md) lowercases the short form ("Unused Commit credits," "additional credits"). Worth picking one form and using it consistently across this page.

> | 2,000 Credits | 200 Credits per user | 20 | 200 × 20 = 4,000 Credits<sup>*</sup> | 2,000 Credits | Each user can use up to 200 AI Credits **as long as the org limit of 2,000 Credits has not been reached.** |

Reviewer: Bold isn't used for general emphasis in Datadog docs, and the bolded text here is a trailing clause rather than a UI element or defined term. The period also sits inside the bold. 

```suggestion
| 2,000 Credits | 200 Credits per user | 20 | 200 × 20 = 4,000 Credits<sup>*</sup> | 2,000 Credits | Each user can use up to 200 AI Credits until the organization-wide limit of 2,000 Credits is reached. |
```

> <sup>*</sup> A configured org limit acts as the ceiling for the account, regardless of the sum of hypothetical per-user limits.

Reviewer: Thanks for removing the duplicate bullet. Two wording leftovers in the footnote itself: "org limit" and "account," where the rest of the page now says "organization." (Suggestion below is Claude-authored.)

```suggestion
<sup>*</sup> A configured organization limit acts as the ceiling for the organization, regardless of the sum of hypothetical per-user limits.
```

> <sup>*</sup> A configured org limit acts as the ceiling for the account, regardless of the sum of hypothetical per-user limits.

Reviewer: This footnote restates the third bullet in **How limits are applied** (line 36) in nearly identical words. We can probably drop one of them (either let the bullet carry the point, or keep the footnote and remove the bullet)

## Usage attribution

> Attribution determines whose limit applies to a given unit of AI usage, and who the AI Credit spend is assigned to.

Reviewer: "attributed to" matches the section heading and reads better than "assigned to." 

```suggestion
Attribution determines whose limit applies to a given unit of AI usage, and who the AI Credit spend is attributed to.
```

| Product | Attribution |
|---|---|

> | [Bits Chat][1] | All Bits Chat usage rolls up to the user email interacting with Bits Chat. |

Reviewer: An email address doesn't interact with a product, so let's reframe so the user is the actor. The same pattern applies to lines 57-59 ("the email address of the user who initiated the action," "who created the workflow"). 

```suggestion
| [Bits Chat][1] | All Bits Chat usage rolls up to the email address of the user interacting with Bits Chat. |
```

> | [Bits Investigation][2] | Manually started investigations roll up to the user email that initiated the action.<br>Automatically triggered investigations (for example, from a monitor firing) roll up to **Autonomous Agents**. |

Reviewer: This row didn't pick up the email-as-actor change that lines 55 and 58 did, so the table is now inconsistent within itself. (Suggestion below is Claude-authored.)

```suggestion
| [Bits Investigation][2] | Manually started investigations roll up to the email address of the user who initiated the action.<br>Automatically triggered investigations (for example, from a monitor firing) roll up to **Autonomous Agents**. |
```

> | [Bits Code][3] | All Bits Code usage rolls up to the user email interacting with Bits Code. |

Reviewer: Same as the row above — this one also didn't pick up the email-as-actor change. (Suggestion below is Claude-authored.)

```suggestion
| [Bits Code][3] | All Bits Code usage rolls up to the email address of the user interacting with Bits Code. |
```

| [Bits Code][3] | All Bits Code usage rolls up to the user email interacting with Bits Code. |

> | [Bits Agent Builder][4] | Agent runs for workflows roll up to the user email that created the workflow.<br>Agent runs for workflows that roll up to service accounts roll up to **Autonomous Agents**. |

Reviewer: "Roll up to" appears twice in this sentence with two different meanings, which makes the rule hard to parse. I *think* this means workflows owned by service accounts. Can you confirm? 

```suggestion
| [Bits Agent Builder][4] | Agent runs for workflows roll up to the email address of the user who created the workflow.<br>Agent runs for workflows created by service accounts roll up to **Autonomous Agents**. |
```

> | [Bits Agent Builder][4] | Agent runs for workflows roll up to the email address of the user who created the workflow.<br>Agent runs for workflows created by service accounts roll up to **Autonomous Agents**. |

Reviewer: Just confirming, since this went in from my suggestion without a reply: my "workflows created by service accounts" wording was a guess at what the original sentence meant. Is that right, or is the actual rule something else (for example, workflows whose *runs* execute as a service account, regardless of who created them)?

If my guess was wrong, this now states the wrong attribution rule, so worth a quick yes/no.
